src/fontra/client/web-components/ui-form.js

```diff
@@ -187,20 +180,9 @@ export class Form extends SimpleElement {
     this._lastValidFieldValues[fieldItem.key] = fieldItem.value;
     const inputElement = document.createElement("input");
     inputElement.type = "number";
-    inputElement.value = getNiceNumber(this._lastValidFieldValues[fieldItem.key]);
-
-    inputElement.onmouseout = (event) => {
-      inputElement.value = getNiceNumber(this._lastValidFieldValues[fieldItem.key]);
-    };
-    inputElement.onmouseover = (event) => {
-      inputElement.value = this._lastValidFieldValues[fieldItem.key];
-    };
-    inputElement.onclick = (event) => {
-      inputElement.value = this._lastValidFieldValues[fieldItem.key];
-    };
-    window.addEventListener("focusout", (event) => {
-      inputElement.value = getNiceNumber(this._lastValidFieldValues[fieldItem.key]);
-    });
+    inputElement.value = fieldItem.numDigits
```

Comment:
You need to explicitly compare with undefined, because zero is a valid input.

Perhaps write a helper function to avoid duplication with the setter.